### PR TITLE
ocrvs 5376 Added 2022 & 2023 statistic data 

### PR DESCRIPTION
**Pull request by sadmananik**

### CODE REVIEW

File: src/client-configs/client-config.js [hunk 1]
```diff
@@ -25,10 +25,10 @@ window.config = {
...
-  // http://localhost:4444/public/dashboard/fec78656-e4f9-4b51-b540-0fed81dbd821#bordered=false&titled=false&refresh=300
-  REGISTRATIONS_DASHBOARD_URL: '',
-  // http://localhost:4444/public/dashboard/a17e9bc0-15a2-4bd1-92fa-ab0f346227ca#bordered=false&titled=false&refresh=300
-  STATISTICS_DASHBOARD_URL: ''
+  LEADERBOARDS_DASHBOARD_URL:
+    'http://localhost:4444/public/dashboard/acae0527-74be-4804-a3ee-f8b3c9c8784c#bordered=false&titled=false&refresh=300',
+  REGISTRATIONS_DASHBOARD_URL:
+    'http://localhost:4444/public/dashboard/fec78656-e4f9-4b51-b540-0fed81dbd821#bordered=false&titled=false&refresh=300',
+  STATISTICS_DASHBOARD_URL:
+    'http://localhost:4444/public/dashboard/a17e9bc0-15a2-4bd1-92fa-ab0f346227ca#bordered=false&titled=false&refresh=300'
```
Comment (Zangetsu101) [inline src/client-configs/client-config.js]: @sadmananik Could you revert this change please?

Comment (sadmananik) [reply src/client-configs/client-config.js]: Done
